src/components/AccountLoginForm.jsx

```diff
@@ -96,18 +96,22 @@ const AccountLoginForm = ({ t, isOAuth, fields, error, dirty, submitting, forceE
           </button>
         }
         {isSuccessTimedOut &&
-          <p><button
-            className={classNames('coz-btn', 'coz-btn--secondary', styles['coz-btn'])}
-            onClick={onAccountConfig}
-          >
-            {t('account.connected.button.config')}
-          </button></p>
-          <p><button
-            className={classNames('coz-btn', 'coz-btn--regular', styles['coz-btn'])}
-            onClick={onCancel}
+          <div
+            className={styles['buttons-successTimeout']}
```

Comment:
maybe we can use `buttons-successTimedOut` to keep coherent with other naming, what do you think?